src/modules/base/Module_v1.sol

```diff
@@ -202,14 +213,14 @@ abstract contract Module_v1 is
 
     function revokeModuleRole(bytes32 role, address target)
         external
-        onlyOrchestratorOwner
```

Comment:
is onlyOrchestratorOwner still a thing then?
If it is essentially the same as onlyModuleRoleAdmin(role) then shouldnt the modifier be replaced everywhere?
I saw it in a few other modules still

Reply:
I get your point, but I do think the modifiers serve two different functions: onlyModuleRoleAdmin() checks for the admin of a specific role, which starts out as the owner, but can also be transferred to somebody else or burned entirely. onlyOrchestratorOwner basically defers authorization to the general admin, without defining a specific role or allowing that kind of mechanism. Does that make sense? Do you think it would be clearer if I renamed the modifier to onlyOchestratorAdmin ?

Reply:
After discussion in the call I renamed all mentions of OrchestratorOwner into OrchestratorAdmin (except in the scripts, where I think having an user be called "owner" still makes sense)